app/assets/stylesheets/recipe_show.scss

```diff
@@ -0,0 +1,70 @@
+@import "colors";
+
+.recipe-info-block {
+  padding-bottom: 20px;
+}
+
+.recipe-info-attr {
+  padding: 0px 5px;
+  border-radius: 5px;
+  background-color: $dark-orange;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+}
+
+.category-name {
+  background-color: $light-orange;
+  border-radius: 5px;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+  padding: 0px 5px;
+}
+
+.ingredient-row {
+  padding-bottom: 20px;
+}
+
+.ingredient-block {
+  background-color: $light-green;
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, border-style, border-width, box-shadow, margin-left, padding